src/components/Collections/CollectionShare/publicShare/manage/publicCollect.tsx

```diff
@@ -23,12 +23,22 @@ export function ManagePublicCollect({
     };
 
     return (
-        <Box>
-            <Typography mb={0.5}>{t('PUBLIC_COLLECT')}</Typography>
-            <PublicShareSwitch
-                checked={publicShareProp?.enableCollect}
-                onChange={handleFileDownloadSetting}
-            />
-        </Box>
+        <Stack>
+            <EnteMenuItem
+                onClick={handleFileDownloadSetting}
+                color="primary"
+                hasSwitch={true}
+                checked={publicShareProp?.enableCollect}>
+                {t('PUBLIC_COLLECT')}
+            </EnteMenuItem>
+            <Typography
+                color="text.secondary"
+                variant="body2"
+                paddingLeft={1.75}
+                paddingTop={0.75}
+                paddingBottom={0.75}>
+                {t('PUBLIC_COLLECT_SUBTEXT')}
+            </Typography>
```

Comment:
In the design,

This is a component https://www.figma.com/file/SYtMyLBs5SAOkTbfMMzhqt/ente-Visual-Design?node-id=7651-44385&t=kvErkPu1cOmUfZJC-0

lets make a component for this in code too